packages/api/src/schema/api-schema.yaml

```diff
@@ -2854,7 +2856,137 @@ components:
         targetSegmentSizeSecs:
           $ref: >-
             #/components/schemas/new-asset-payload/properties/targetSegmentSizeSecs
-
+    # AI Generate payloads. Keep in mind that these use snake_case instead of camelCase since
+    # they implement the same interface as the AI Gateway Livepeer node.
+    text-to-image-payload:
+      type: object
+      required:
+        - prompt
```

Comment:
Any plan to add a check in the validator for the prompt for potentially harmful content?

Reply:
Hmm not yet! I wonder if we should ever do that though. What we currently have is the `safety_check` which I believe checks if the output is NSFW (not sure if it returns an error or just flags the response `nsfw` field)